pom.xml

```diff
@@ -1797,6 +1797,9 @@
 
                 <!-- tests should be updated -->
                 <exclude>**/AbstractJavadocCheckTest.class</exclude>
+
+                <!-- due to https://github.com/checkstyle/checkstyle/pull/13193 All the test needs to be in single line -->
+                <exclude>**/OneStatementPerLineCheckTest.class</exclude>
```

Comment:
I do not understand why you need this

Reply:
If I use `verifyWithInlineConfigParser()` method I will get the error config comment not specified properly because we want all the code in a single line.

Therefore I have to use `verify()` method and if i use verify method then at the end of mvn clean verify I got the error like 
```
[INFO] Scanning classes for violations...
[ERROR] Forbidden method invocation: com.puppycrawl.tools.checkstyle.AbstractModuleTestSupport#verify(com.puppycrawl.tools.checkstyle.api.Configuration, java.lang.String, java.lang.String[]) [Use inline config parser instead.]
[ERROR]   in com.puppycrawl.tools.checkstyle.checks.coding.OneStatementPerLineCheckTest (OneStatementPerLineCheckTest.java:139)
[ERROR] Scanned 3471 class file(s) for forbidden API invocations (in 1.19s), 1 error(s).
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  10:47 min
[INFO] Finished at: 2023-06-19T00:57:15+05:30
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal de.thetaphi:forbiddenapis:3.4:testCheck (forbiddenapis-test) on project checkstyle: Check for forbidden API calls failed, see log. -> [Help 1]

```

Reply:
Please move this config to be above `<!-- all bellow until`

Reply:
done

Reply:
ohh. I got your point here https://github.com/checkstyle/checkstyle/pull/13193#discussion_r1233361346 will take care of this stuff https://github.com/checkstyle/checkstyle/pull/13193#discussion_r1233366516